### ISSUE TITLE
mean teacher's ap is lower than paper 

### ISSUE THREAD

**Issue by KaoJZhu** [PointsCoder/ONCE_Benchmark#29]
mean teacher result ,which  test on validation set and train on small unlabel set.
|AP@50       |overall     |0-30m       |30-50m      |50m-inf     |
|Vehicle     |73.31       |84.82       |68.15       |52.92       |
|Pedestrian  |31.88       |36.56       |27.27       |18.93       |
|Cyclist     |61.41       |73.01       |54.92       |37.50       |
|mAP         |55.53       |64.80       |50.12       |36.45       |

the ap is lower than the paper, especially the distance is 0-30m

**Comment by KaoJZhu** [comment]
baseline
|AP@50       |overall     |0-30m       |30-50m      |50m-inf     |
|Car         |73.10       |83.01       |67.48       |52.16       |
|Bus         |72.18       |83.42       |75.62       |46.33       |
|Truck       |38.51       |58.01       |37.03       |17.79       |
|Pedestrian  |28.38       |31.61       |25.40       |17.15       |
|Cyclist     |58.21       |70.30       |52.39       |34.30       |
|mAP         |54.07       |65.27       |51.59       |33.54       |

mean teacher
|AP@50       |overall     |0-30m       |30-50m      |50m-inf     |
|Car         |75.00       |85.13       |68.35       |55.60       |
|Bus         |72.09       |83.46       |74.90       |45.53       |
|Truck       |34.03       |50.46       |31.72       |14.38       |
|Pedestrian  |31.88       |36.56       |27.27       |18.93       |
|Cyclist     |61.41       |73.01       |54.92       |37.50       |
|mAP         |54.88       |65.72       |51.43       |34.39       |
The ap of the truck decrease 8 points. Did you meet this result?
